### ISSUE TITLE
Error happens when newly installing slide merge

### ISSUE THREAD

**Issue by kikyou0797** [brucemcpherson/desktopliberation#9]
When I want to install slide merge to a new pc (different account),   it said "needs your permission to run." After I chose my google account, a error shows up as " Sign in with Google temporarily disabled for this app & This app has not been verified yet by Google in order to use Google Sign In". Wish this can be fixed.

**Comment by oshliaer** [comment]
It seems this app temporary closed for public use.

**Comment by brucemcpherson** [comment]
Yes - I'm sorry about this. Google introduced various retrospective verfication processes and I never found the time to put all my add-ons through them all, as it's quite laborious and time intensive. I announced a couple of months ago, https://plus.google.com/112066118406610145134/posts/13SctmRc2Tt ,
that I was stopping supporting or working on add-ons - it's just too much effort for something that generates no revenue. I have made them all open source in case anyone wants to pick them up as add-ons, or indeed use them within your own organization.
